.github/workflows/links.yml

```
@@ -0,0 +1,46 @@
+        uses: peter-evans/create-issue-from-file@v5
+        with:
+          title: Link Checker Report
+          content-filepath: ./lychee/out.md
+          labels: report, automated issue
+
+      - name: Save lychee cache
+        uses: actions/cache/save@v4
+        if: always()
+        with:
+          path: .lycheecache
+          key: ${{ steps.restore-cache.outputs.cache-primary-key }}
```

actionlint (1.7.4)

property "restore-cache" is not defined in object type {lychee: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)